File: java/8.3/example3.java
import java.util.Scanner;

public class example3 {
    public static void main(String[] args) {
        Integer number1 = 5;
        System.out.println("Enter a number");
        Scanner scanner = new Scanner(System.in);
        Integer number2 = Integer.parseInt(scanner.nextLine());
        Integer result = number1 * number2;
        System.out.println(result);
    }
}

File: rust/src/8.3/example3.rs
fn main() {
    let number1 = 5;
    let mut number2s = String::new();
    println!("Enter a number");
    std::io::stdin().read_line(&mut number2s).unwrap();
    let number2 = number2s.trim().parse::<i32>().unwrap();
    let result = number1 * number2;
    println!("The rsesult is {}", result);
}